x-pack/plugins/security/server/authentication/index.ts

```diff
@@ -134,6 +135,12 @@ export async function setupAuthentication({
     login: authenticator.login.bind(authenticator),
     logout: authenticator.logout.bind(authenticator),
     getCurrentUser,
+    createApiKey: (request: KibanaRequest, body: CreateApiKeyOptions['body']) =>
```

Comment:
__issue:__ please, don't forget to update signature of [PluginSetupContract](https://github.com/elastic/kibana/blob/91e1d9c712460d18a90d0a1411eef48d7ef41f0f/x-pack/plugins/security/server/plugin.ts#L34)